src/test/java/com/amazon/opendistroforelasticsearch/security/test/AbstractSecurityUnitTest.java

```diff
@@ -307,7 +307,11 @@ protected String getType() {
      * @return boolean flag indicating if transport settings are defined
      */
     protected boolean hasCustomTransportSettings(Settings customSettings) {
-        // Note: current only doing this for PEMCERT settings
-        return customSettings.get(SSLConfigConstants.OPENDISTRO_SECURITY_SSL_TRANSPORT_PEMCERT_FILEPATH) != null;
+        if (customSettings.get(SSLConfigConstants.OPENDISTRO_SECURITY_SSL_TRANSPORT_EXTENDED_KEY_USAGE_ENABLED) != null) {
```

Comment:
There is no need to check for `null`, use `Boolean.parseBoolean()`, no need for a separate `if` (use OR).